air-script/tests/selectors/selectors_combine_with_list_comprehensions.air

```diff
@@ -0,0 +1,59 @@
+def SelectorsAir
+
+trace_columns {
+    main: [s[3], a, b, c],
+}
+
+public_inputs {
+    stack_inputs: [1],
+}
+
+boundary_constraints {
+    enf c.first = 0;
+}
+
+# Note: we want the following constraints values to be equivalent:
```

Comment:
I don't fully understand what this comment means at first glance. What are we testing/expecting here?